src/Collector/MethodCall/MethodCallArgTypesCollector.php

```diff
@@ -34,7 +34,13 @@ public function getNodeType(): string
      */
     public function processNode(Node $node, Scope $scope): ?array
     {
-        if ($node->getArgs() === []) {
+        if ($node->getArgs() === [] || !$node->name instanceof Node\Identifier) {
```

Comment:
when `$node->isFirstClassCallable()`, it should return null early, as `$node->getArgs()` may cause assert warning

Reply:
fixed thanks (also in another spot, not related to the PR)